modules/mobile/app/models/mobile/v0/adapters/discovery.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Adapters
```

Comment:
I think the implementation is good just not sure it's an 'adapter' 😄 appointments and claims meet that description because they are taking objects with one shape and mapping their data to another. In this case we're taking a couple args and returning new data. Also wonder if the below logic could move to a model then the model's constructor would assign the correct values to its attributes when it's instantiated. e.g.

```ruby
discovery = Discovery.new(build_number, os)
render json: DiscoverySerializer.new(discovery)
```

Reply:
Gotcha, that makes sense. I'll move that stuff